free-programming-books.md

```diff
@@ -471,7 +471,7 @@
 * [Bayesian Reasoning and Machine Learning](http://web4.cs.ucl.ac.uk/staff/D.Barber/pmwiki/pmwiki.php?n=Brml.HomePage)
 * [Deep Learning](http://www.deeplearningbook.org) - Ian Goodfellow, Yoshua Bengio and Aaron Courville
 * [Deep Learning with PyTorch](https://pytorch.org/assets/deep-learning/Deep-Learning-with-PyTorch.pdf) - Eli Stevens, Luca Antiga, Thomas Viehmann (PDF)
-* [Dive into Deep Learning](http://d2l.ai)
+* [Dive into Deep Learning](http://d2l.ai) - Aston Zhang et al (HTML) [PDF](https://d2l.ai/d2l-en.pdf)
```

Comment:
good to add creators, but we don't usually list seperate links for particular formats

Reply:
OK. 

I saw a format like that though.

Reply:
Occasionally there's a good reason. not here.